From: Simon Horman <horms+renesas@verge.net.au>
Date: Thu, 3 Mar 2016 10:28:50 +0900
Subject: ata: sata_rcar: Use ARCH_RENESAS
References: bsc#977572
Patch-mainline: v4.6-rc1
Git-commit: 51b1b2867b81450bd198b691c70dccb582e961ff

Make use of ARCH_RENESAS in place of ARCH_SHMOBILE.

This is part of an ongoing process to migrate from ARCH_SHMOBILE to
ARCH_RENESAS the motivation for which being that RENESAS seems to be a more
appropriate name than SHMOBILE for the majority of Renesas ARM based SoCs.

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
Signed-off-by: Tejun Heo <tj@kernel.org>
Acked-by: Geert Uytterhoeven <geert+renesas@glider.be>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/ata/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ata/Kconfig b/drivers/ata/Kconfig
index 9c15828..5083f85 100644
--- a/drivers/ata/Kconfig
+++ b/drivers/ata/Kconfig
@@ -364,7 +364,7 @@ config SATA_PROMISE
 
 config SATA_RCAR
 	tristate "Renesas R-Car SATA support"
-	depends on ARCH_SHMOBILE || COMPILE_TEST
+	depends on ARCH_RENESAS || COMPILE_TEST
 	help
 	  This option enables support for Renesas R-Car Serial ATA.
 
-- 
1.8.5.6

